databuilder/ehrql-tutorial-examples/project.yaml

```diff
@@ -4,8 +4,56 @@ expectations:
   population_size: 3000
 
 actions:
-  extract_minimal_population:
-    run: databuilder:v0 generate-dataset "./1a_minimal_dataset_definition.py" --dummy-tables "example-data/minimal" --output "outputs/1a_minimal_dataset_definition.csv"
+  extract_1a_minimal_population:
+    run: databuilder:v0.57.0 generate-dataset "./1a_minimal_dataset_definition.py" --dummy-tables "example-data/minimal" --output "outputs/1a_minimal_dataset_definition.csv"
```

Comment:
Should this not be `databuilder:dev`

Reply:
Perhaps! But this is slightly trickier than it looks.

It's deliberately tied to the Python `requirements.dev.txt` for now, to ensure consistency, when running the examples.

1. It's not possible to just specify an equivalent "latest v0 version" for those requirements: we specify hash checks there, which requires that we know what the hash is, and have to keep that updated.
2. If we instead specify a vague tag like `v0` or `dev`, then I assume that this might try and run with an older `v0` or `dev` image, if someone has an older `v0` image already on their machine and doesn't update. If we're at v1 and ensuring no breaking changes, then it might be more reasonable to pin to v1.

In the meantime, we could add an automated workflow in future to keep the images in sync with the requirements specified for Python.